telegram/ext/dispatcher.py

```diff
@@ -227,8 +226,8 @@ def stop(self):
                 sleep(0.1)
             self.__stop_event.clear()
 
-        # async threads must be join()ed only after the dispatcher thread was joined,
-        # otherwise we can still have new async threads dispatched
+# async threads must be join()ed only after the dispatcher thread was joined,
+# otherwise we can still have new async threads dispatched
```

Comment:
Oh, that must have been the pre-commit hooks. I don't remember making the change.